Dockerfile

```diff
@@ -73,19 +54,24 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=true \
 RUN apk add --no-cache --virtual=.run-deps bash git git-lfs openssh
 COPY cli ./
 
+RUN apk add --no-cache --virtual=.build-deps build-base
+
+# DL3013: Pin versions in pip.
 # hadolint ignore=DL3013
-RUN apk add --no-cache --virtual=.build-deps build-base && \
-     SEMGREP_SKIP_BIN=true pip install /semgrep && \
-     # running this pre-compiles some python files for faster startup times
-     semgrep --version && \
-     apk del .build-deps && \
-     mkdir -p /tmp/.cache
+RUN SEMGREP_SKIP_BIN=true pip install /semgrep
```

Comment:
@mjambon maybe that's why the docker image grew from 64MB to 127MB? That's why we need to "consolidate"
and put the RUN commands together?
cc @brendongo and @underyx

Reply:
Yep. File deletions only reduce image size if they're in the same RUN instruction that created them.

Reply:
wow, really? Why? So how can we save space by deleting all /semgrep/tests?
Be more precise at the COPY cli ./?
and ignore tests/ subdir?